WikiWatershed/model-my-watershed
# Replace precipitation dropdown with slider

How to test:
1. Rebuild JS bundle
2. Verify slider control works on scenario tabs
3. Verify duplicating scenarios retains slider value

Connects #272
